docs/languages.rst

```diff
@@ -75,4 +75,5 @@ A detailed instruction on how to train the model yourself can be found in
 A trained model from Chinese Wikipedia Dump and Baidu Baike can be `crownpku <https://github.com/crownpku>`_  's
 `blogpost <http://www.crownpku.com/2017/07/27/%E7%94%A8Rasa_NLU%E6%9E%84%E5%BB%BA%E8%87%AA%E5%B7%B1%E7%9A%84%E4%B8%AD%E6%96%87NLU%E7%B3%BB%E7%BB%9F.html>`_.
 
+Please rename ``total_word_feature_extractor_zh.dat`` to `total_word_feature_extractor.dat` and put it in ``data`` folder so that the MITIE can load it.
```

Comment:
why do you think it is better to rename it?

Reply:
The default model name of mitie is ```total_word_feature_extractor.dat``` but the user can config it in the config file. It seems not a problem.

Reply:
I think I'd rather keep the original file name and instead add a sentence about the configuration and maybe even include it in the docs.

Please also merge in master, the yml files have been fixed in a separate PR.